ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -217,9 +218,9 @@ mkCompl
             "line " <> printOutputable (srcLocLine loc) <> ", column " <> printOutputable (srcLocCol loc)
 
 
-mkAdditionalEditsCommand :: PluginId -> ExtendImport -> Command
-mkAdditionalEditsCommand pId edits =
-  mkLspCommand pId (CommandId extendImportCommandId) "extend import" (Just [toJSON edits])
+mkAdditionalEditsCommand :: Maybe PluginId -> ExtendImport -> Maybe Command
+mkAdditionalEditsCommand (Just pId) edits = Just $ mkLspCommand pId (CommandId extendImportCommandId) "extend import" (Just [toJSON edits])
```

Comment:
Would it make sense to introduce an abstraction for this dynamic lookup now?
Something like:
```
genLspCommand :: IdePlugins a -> CommandId -> Maybe (Text -> ... -> Command)
genLspCommand = fmap (mkLspCommand ...) (lookupPluginId ...)
```